app/actors/sufia/default_admin_set_actor.rb

```diff
@@ -25,7 +25,7 @@ def ensure_admin_set_attribute!(attributes)
       end
 
       def ensure_permission_template!(admin_set_id:)
-        Sufia::PermissionTemplate.find_or_create_by!(admin_set_id: admin_set_id)
+        Sufia::PermissionTemplate.find_by(admin_set_id: admin_set_id) || create_permission_template!(admin_set_id: admin_set_id)
```

Comment:
This would be a change in behavior, I believe. The PR that added this code allows it to create a PT for any AS: https://github.com/projecthydra/sufia/commit/9adea5d61d98f1f63779eaa41774e88d70529af8#diff-eec0400feb42a8dea093932c1fe1bc3a